python/tank/deploy/tank_commands/core_upgrade.py

```diff
@@ -15,36 +15,35 @@
 import sys
```

Comment:
File still needs to be renamed, but I didn't do it right away because I was afraid it would mess the diff in Github.